grouper/background.py

```diff
@@ -7,10 +7,18 @@
 from sqlalchemy import and_
 from sqlalchemy.exc import OperationalError
 
-from grouper.email_util import notify_edge_expiration, process_async_emails
-from grouper.model_soup import Group, GroupEdge
+from grouper.constants import PERMISSION_AUDITOR
+from grouper.email_util import (notify_edge_expiration, notify_nonauditor_flagged,
```

Comment:
if we're going to break out to multiple lines, I'd prefer we have one item per line. But @herb feel free to digress.